codegen/src/main/java/software/amazon/awssdk/codegen/AddShapes.java

```diff
@@ -124,8 +124,6 @@ protected final ShapeModel generateShapeModel(String javaClassName, String shape
         final List<String> enumValues = shape.getEnumValues();
         if (enumValues != null && !enumValues.isEmpty()) {
             for (String enumValue : enumValues) {
-                // TODO handle useRealName if explicitly mentioned in
```

Comment:
did we already finish this todo?

Reply:
This TODO is from [v1](https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-code-generator/src/main/java/com/amazonaws/codegen/AddShapes.java#L136) which was added 3 years back. I think we never prioritized it. So removed it as its not specific to v2 and not a blocker for GA.